lib/shared/stats.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+
+module.exports = {
+
+    /**
+     * Start time measurement
+     * @returns {float} t variable for tracking time
+     */
+    startTime() {
+        return process.hrtime();
+    },
+
+    /**
+     * End time measurement
+     * @param {float} t Variable for tracking time
+     * @returns {void}
```

Comment:
void?

Reply:
This should be something like this?
```suggestion
     * @returns {float} The measured time in milliseconds
```

Reply:
Yep.